config/presets/identity/email-password-interaction-policy.json

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
We might want some `"comment": "…"` lines in the configs.

Reply:
Done